documentation/logs.md

```diff
@@ -0,0 +1,152 @@
+and `trace` level logs to a `logs` directory in the Bolt project:
+
+```yaml
+# bolt-project.yaml
+name: lotsalogs
+
+log:
+  "logs/trace.log":
+    append: false
+    level: trace
+  "logs/info.log":
+    append: false
+    level: info
+```
+
+Set the `append` key to `false` if you want to overwrite the file on each run,
+or `true` if you want to append to the existing file.
+ 
+> **Note**: The directory you set for logging must exist, and Bolt must have
+> permission to write to it.
+
+## Log levels
+
+### `trace`
+
+Trace logs contain the most detailed information for actions run on individual
+targets. For example, the logs include each command run on a target, each step
+of compiling a manifest for an apply block, and information about establishing
+connections with targets.
+
+Trace logs are useful when you're trying to troubleshoot a problem with Bolt.
+They contain information about what exactly Bolt is doing when it runs an action
+on each target.
+
+### `debug`
+
+Debug logs contain information for target-specific steps and detailed
+information about where Bolt is loading data from. For example, you'll find
+information on how Bolt locates the project to load, where different content is
+loaded from, and information about actions run on each individual target.
+
+Debug logs are useful for troubleshooting a problem with a plan or the way
+you're using Bolt. You can use debug logs for a high-level understanding of what
+actions Bolt is running on individual targets, or to find out where Bolt is
+loading data from.
+
+### `info`
+
+Info logs give you a high-level overview of what Bolt is doing. For example, an
+info log contains information about which Bolt project is loaded, when different
+actions run on targets start and finish, and results from actions run on
+targets.
+
+Info logs are useful when you want to see the results for an action in a Bolt
+execution. They're also useful for a high-level understanding of what Bolt is
+doing.
+
+### `warn`
+
+Warn logs contain warnings about deprecations and potentially harmful situations
+that might affect your Bolt run, even though they don't prevent Bolt from
+executing. For example, Bolt warns you if your inventory file includes an
+unsupported transport configuration that might result in Bolt connecting to
+targets in a way that you don’t expect.
+
+Bolt always prints warnings to the console.
+
+Use a warn log if you're trying to find out if you're using Bolt in a way that
+might result in unexpected behavior, or to see if you're using a deprecated
+feature.
+
+### `error`
+
+Error logs contain messages for errors that Bolt encountered during execution.
+For example, Bolt prints an error if a PuppetDB query fails.
+
+Bolt always prints errors to the console.
+
+Use an error log if you want to see errors that Bolt raised during a Bolt run.
+
+## Using `verbose` logging
+
+Bolt's `run`, `file upload`, and `file download` commands include the
```

Comment:
```suggestion
Bolt's `(command | plan | script | task) run` and `file (upload | dowload)` commands include the
```

Reply:
The original wording just made it sound like there was a `bolt run` command - I'm not sure if this is clearer or not, but some other syntax might be more useufl.

Reply:
Okay, let me take another swing at this